crates/backend/src/codegen.rs

```diff
@@ -223,6 +223,11 @@ impl ToTokens for ast::Struct {
         let free_fn = Ident::new(&shared::free_function(&name_str), Span::call_site());
         let unwrap_fn = Ident::new(&shared::unwrap_function(&name_str), Span::call_site());
         let wasm_bindgen = &self.wasm_bindgen;
+        let maybe_no_coverage = if cfg!(wasm_bindgen_unstable_test_coverage) {
+            Some(quote! { #[coverage(off)] })
+        } else {
+            None
+        };
```

Comment:
There are 6 copies of this logic in this file. Maybe this could be moved out into a constant or function?

Reply:
Done.